api/lib/spree/api.rb

```diff
@@ -10,3 +10,7 @@ module Api
 
 require 'spree/api/api_dependencies'
 require 'spree/api/engine'
+
+Dir["#{File.dirname(__FILE__)}/factories/**"].each do |f|
```

Comment:
@aleksandarpetrushev why code duplication? you can just require `testing_support/factories.rb` here

Reply:
I seem to get NameError: uninitialized constant Spree::Webhooks

Reply:
create a `api/lib/spree/webhooks.rb` and define the module there, and also you can move the requirement there

Reply:
I would also need to define ::Events and ::Subsciber modules as well. And any others added in the future will need to be defined

Reply:
Actually looks like we can not do it that way as well